server/routes/approvingLicences/handlers/approvalCases.test.ts

```diff
@@ -92,10 +128,31 @@ describe('Route Handlers - Approval - case list', () => {
             name: 'Bob Smith',
             prisonerNumber: 'A1234AA',
             releaseDate: '01 May 2022',
+            releaseDateLabel: 'Confirmed release date',
             probationPractitioner: {
               name: 'Walter White',
             },
           },
+          {
+            licenceId: 2,
+            name: 'Joe Bloggs',
+            prisonerNumber: 'A1234AB',
+            probationPractitioner: {
+              name: 'Thor',
+            },
+            releaseDate: '01 May 2022',
+            releaseDateLabel: 'CRD',
+          },
+          {
+            licenceId: 3,
+            name: 'Harvey Smith',
+            prisonerNumber: 'A1234AC',
+            probationPractitioner: {
+              name: 'Walter Black',
+            },
+            releaseDate: '01 May 2022',
+            releaseDateLabel: 'CRD',
```

Comment:
Harvey Smith has a 'conditional release date' so the releaseDateLabel should be CRD